includes/Checker/Check.php

```diff
@@ -15,6 +15,30 @@
  * @since n.e.x.t
  */
 interface Check {
+	/**
+	 * Flag for stable checks.
+	 *
+	 * @since n.e.x.t
+	 * @var int
+	 */
+	const STABILITY_STABLE = 1;
+
+	/**
+	 * Flag for experimental checks.
+	 *
+	 * @since n.e.x.t
+	 * @var int
+	 */
+	const STABILITY_EXPERIMENTAL = 2;
```

Comment:
I don't think these need to be bitwise flags, as that wouldn't bring a benefit here. We can use simple strings to improve debugging (seeing a string `STABLE` will be more helpful than seeing `1`).

The constants should only exist to not rely on hard-coded strings.
```suggestion
	/**
	 * Stability value for stable checks.
	 *
	 * @since n.e.x.t
	 * @var string
	 */
	const STABILITY_STABLE = 'STABLE';

	/**
	 * Stability value for experimental checks.
	 *
	 * @since n.e.x.t
	 * @var string
	 */
	const STABILITY_EXPERIMENTAL = 'EXPERIMENTAL';
```

Reply:
Thanks @felixarntz I've made those updates